src/front/js/pages/home.js

```diff
@@ -1,26 +1,17 @@
 import React, { useContext } from "react";
-import { Context } from "../store/appContext";
-import rigoImageUrl from "../../img/rigo-baby.jpg";
+// import { Context } from "../store/appContext";
 import "../../styles/home.css";
 
-export const Home = () => {
-	const { store, actions } = useContext(Context);
+import Testimonials from "../component/Testimonials";
+import Pricing from "../component/Pricing";
+import Header from "../component/Header";
 
-	return (
-		<div className="text-center mt-5">
-			<h1>Hello Rigo!!</h1>
-			<p>
-				<img src={rigoImageUrl} />
-			</p>
-			<div className="alert alert-info">
-				{store.message || "Loading message from the backend (make sure your python backend is running)..."}
-			</div>
-			<p>
-				This boilerplate comes with lots of documentation:{" "}
-				<a href="https://github.com/4GeeksAcademy/react-flask-hello/tree/95e0540bd1422249c3004f149825285118594325/docs">
-					Read documentation
-				</a>
-			</p>
-		</div>
-	);
+export const Home = () => {
+  return (
+    <React.Fragment>
+      <Header />
+      <Pricing />
+      <Testimonials />
```

Comment:
In this case, on the other hand, I don't know if dividing a page into 3 components like this would be the best idea... We're not using them any elsewhere in our project, right?

And also... They look so well together! Maybe they're actually meant to be just one beautiful component! 🌞
What do you think?